.travis.yml

```diff
@@ -12,3 +24,6 @@ script:
   - bundle exec rubocop --version
   - bundle exec rubocop
   - bundle exec env REDIS_URL=redis://localhost:6379 rake test
+  - nutcracker --test-conf --conf-file=.travis/nutcracker.yml
+  - nutcracker             --conf-file=.travis/nutcracker.yml &
+  - bundle exec env REDIS_URL=redis://localhost:22121 rake test
```

Comment:
I am finding Travis to be a more pliable medium than I expected, which is gratifying.